tests/molecular/topology_graphs/vertex/cases/cof/fixtures/cof1.py

```diff
@@ -23,12 +23,10 @@ def cof1(cls, id, position, aligner_edge, cell):
 def unaligned_cof1(cls, id, position, aligner_edge, cell):
     return CaseData(
         vertex=stk.cof.UnaligningVertex(
-            vertex=cls(
-                id=id,
-                position=position,
-                aligner_edge=aligner_edge,
-                cell=cell,
-            )
+            id=id,
+            position=position,
+            aligner_edge=aligner_edge,
+            cell=cell,
```

Comment:
here the cls fixture is no longer being used here `unaliged_cof` fixutre -> I assume we can remove it?

Reply:
can we remove this whole fixture?